packages/ui/src/components/editor/panels/Viewport/tools/GizmoTool.tsx

```diff
@@ -44,66 +44,125 @@ function Placer() {
-              )}
-              iconContainerClassName="m-0"
-              startIcon={<TbVector />}
-              title="Translate (key W)"
-              onClick={() => setTransformMode(TransformMode.translate)}
-            />
-          </Tooltip>
-          <Tooltip title={t('editor:toolbar.gizmo.rotate')} position={'right center'}>
-            <Button
-              variant="transparent"
-              className={twMerge(
-                'border-b border-b-theme-primary p-2 text-[#A3A3A3]',
-                transformMode === TransformMode.rotate && 'bg-theme-highlight text-white'
-              )}
-              iconContainerClassName="m-0"
-              startIcon={<TbRefresh />}
-              title="Rotate (key E)"
-              onClick={() => setTransformMode(TransformMode.rotate)}
-            />
-          </Tooltip>
-          <Tooltip title={t('editor:toolbar.gizmo.scale')} position={'right center'}>
-            <Button
-              variant="transparent"
-              className={twMerge(
-                'p-2 text-[#A3A3A3]',
-                transformMode === TransformMode.scale && 'bg-theme-highlight text-white'
-              )}
-              iconContainerClassName="m-0"
-              startIcon={<TbWindowMaximize />}
-              title="Scale (key R)"
-              onClick={() => setTransformMode(TransformMode.scale)}
-            />
-          </Tooltip>
-        </div>
-      </Tooltip>
+    <div
+      ref={gizmoRef}
+      className={`absolute z-[5] flex flex-col items-center rounded-lg bg-black p-2`}
+      style={{
+        left: `${position.x}px`,
+        top: `${position.y}px`
+      }}
+    >
+      <div className={`z-[6] ${isDragging ? 'cursor-grabbing' : 'cursor-grab'}`} onMouseDown={handleMouseDown}>
+        <Placer />
+      </div>
+      <div className="mt-2 flex flex-col rounded bg-theme-surface-main">
+        <Tooltip title={t('editor:toolbar.gizmo.pointer')} position={'right center'}>
+          <Button
+            variant="transparent"
+            className={twMerge(
+              'border-b border-b-theme-primary p-2 text-[#A3A3A3]',
+              transformMode === TransformMode.pointer && 'bg-theme-highlight text-white'
+            )}
+            iconContainerClassName="m-0"
+            startIcon={<TbPointer />}
+            onClick={() => {
+              setTransformMode(TransformMode.pointer)
+              EditorControlFunctions.replaceSelection([])
+            }}
+          />
+        </Tooltip>
+        <Tooltip title={t('editor:toolbar.gizmo.translate')} position={'right center'}>
+          <Button
+            variant="transparent"
+            className={twMerge(
+              'border-b border-b-theme-primary p-2 text-[#A3A3A3]',
+              transformMode === TransformMode.translate && 'bg-theme-highlight text-white'
+            )}
+            iconContainerClassName="m-0"
+            startIcon={<TbVector />}
+            title="Translate (key W)"
```

Comment:
the titles on the buttons are redundant now with the usage of tooltips (and also incorrect). can you remove them on all these 4 buttons?

![image](https://github.com/user-attachments/assets/4be5f056-e549-4eb6-ac61-4a39d9d5b9db)